client/my-sites/plugins/plugins-browser/index.jsx

```diff
@@ -262,10 +265,24 @@ const PluginsBrowser = React.createClass( {
 			return;
 		}
 
+		const suggestedSearches = [
+			this.props.translate( 'Engagement', { context: 'Plugins suggested search term' } ),
+			this.props.translate( 'Security', { context: 'Plugins suggested search term' } ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **34** times:
`translate( 'Security', { context: 'settings screen'} )` *ES Score: 17*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/plugins-suggested-searches)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

:information_source: This **context** is really long. Are you sure you don't want to use a translator comment instead?